projects/1.18/assets/fat-chicken/fatchicken/lang/zh_cn.json

```
@@ -0,0 +1,14 @@
+{
+  "item.fatchicken.fatchicken_raw": "大号生鸡肉",
+  "block.fatchicken.fatchicken": "大号烤鸡",
+  "item.fatchicken.fatpig_raw": "大号生猪肉",
+  "block.fatchicken.fatpig": "大号烤猪排",
+  "item.fatchicken.fatfish_raw": "大号生鲑鱼",

```

```suggestion
  "item.fatchicken.fatfish_raw": "大条生鲑鱼",
```
确定是`salmon`而不是`fish`吗

---

是`salmon`。因为`entity.fatchicken.fatfish`就是甜浆果喂鲑鱼喂出来的。